packages/env/lib/build-docker-compose-config.js

```diff
@@ -241,6 +243,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 					...dbEnv.development,
 					WP_TESTS_DIR: '/wordpress-phpunit',
 				},
+				extra_hosts: [ 'host.docker.internal:host-gateway' ],
```

Comment:
This ultimately becomes

```
    extra_hosts:
      - 'host.docker.internal:host-gateway'
```

in the yml file. Not sure if the extra quotations marks will matter

Reply:
Looking at the YAML spec it seems that using single-quotes like this just clarifies that the type is a string. It's useful in cases where they may be ambiguity, so once parsed, `docker-compose` won't see any quotes.